src/software/ai/hl/stp/stp.h

```diff
@@ -25,7 +26,7 @@ class STP : public HL
      * The default value is 0
      */
     explicit STP(std::function<std::unique_ptr<Play>()> default_play_constructor,
-                 long random_seed = 0);
+                 std::shared_ptr<const AIControlConfig> config, long random_seed = 0);
```

Comment:
Also should be control_config here and subsequent areas.

Reply:
changed